source/opt/local_redundancy_elimination.cpp

```diff
@@ -58,6 +65,22 @@ bool LocalRedundancyEliminationPass::EliminateRedundanciesInBB(
       context()->ReplaceAllUsesWith(inst->result_id(), candidate.first->second);
       context()->KillInst(inst);
       modified = true;
+    } else if (inst->GetOpenCL100DebugOpcode() ==
+                   OpenCLDebugInfo100DebugDeclare ||
+               inst->GetOpenCL100DebugOpcode() ==
+                   OpenCLDebugInfo100DebugValue) {
+      uint32_t var_id =
```

Comment:
There might be a better way to handle this.  Why is the result of `GetValueNumber` different for the two redundant DebugValue instructions?  I would expect that we should change value number to make them the same.

Reply:
done